app/views/layouts/_header.html.erb

```diff
@@ -22,6 +22,11 @@
             <% if current_user.present? %>
               <%= link_to "Sign out", signout_path, class: "govuk-header__link" %>
               <%= link_to "(#{current_user['info']['first_name']} #{current_user['info']['last_name']})", "#{Settings.dfe_signin.profile}", class: "govuk-header__link" %>
+              <% if current_user["admin"] %>
+                <li class="govuk-header__navigation-item">
```

Comment:
This has created a nested list item. You’ll need to move it outside of the parent li